client/asset/eth/eth.go

```diff
@@ -338,15 +337,92 @@ func (eth *ExchangeWallet) OwnsAddress(address string) (bool, error) {
 
 // Balance returns the total available funds in the account. The eth node
 // returns balances in wei. Those are flored and stored as gwei, or 1e9 wei.
```

Comment:
We should doc that unconfirmed incoming value other than our own redeem and refund txns is not reflected.